Microsoft.Toolkit.Uwp.SampleApp/SamplePages/OneDrive Service/OneDrivePage.xaml.cs

```diff
@@ -46,6 +47,7 @@ public OneDrivePage()
             }
 
             Shell.Current.DisplayWaitRing = true;
+            bool succeeded = true;
```

Comment:
Why not assume `false` as start value and only change to `true` in the end of the process? Seems to make more sense and simpler to implement (would only require a single place to change the value)